app/javascript/authentication/token.ts

```diff
@@ -13,7 +13,7 @@ const getStorage = () => {
 }
 
 const getAuthHeaders = (): AuthHeaders | AxiosHeaders | undefined => {
-  const headers: string = getStorage()[ACCESS_TOKEN_LOCAL_STORAGE_KEY]
+  const headers: string = getStorage().getItem(ACCESS_TOKEN_LOCAL_STORAGE_KEY)
```

Comment:
note: I switched to the recommended `getItem` syntax to make it easier to mock for tests.